proposals/template.md

```diff
@@ -41,8 +36,28 @@ Process:
 
 ...
 
-## Other Considerations
-> >
+-----------------------
+
+## II. Proposal Details
+
+...(Detailed information here)...
+
+-----------------------
+
+## III. Additional Information
+
+<!-- Note: Author may delete any headers in this section which are not relevant. -->
+
+### Which layer(s) of the Singer ecosystem does this proposal directly touch?
+
+Select all that apply:
+
+- [ ] Singer Specification - required capabilities and behaviors
+- [ ] Singer Specification - optional capabilities and behaviors
+- [ ] Singer best practices and other guidance
+- [x] **Singer Working Group - practices and procedures**
+- [ ] Singer documentation (Other)
+
 ### Are there any downsides to this change?
```

Comment:
@dmosorast - Thanks for this. I've updated the prompt to be more direct.

Reply:
@dmosorast - Re:

> Feels like there's a balance we need to strike here between being too prescriptive and being too loose. We'll learn how best to do this overtime so it may be good to keep it more open...

I agree. And I've added a section to allow small edits/tweaks to incorporate our learnings over time via simple PR approval (along with a required changelog per doc) rather than requiring a full new process. While we certainly would want this "Edit" process to be used to slip in huge alterations, I think at least _some_ flexibility is important to give us more confidence we can continue to to iterate based on learnings.